ChatApplication/ViewModels/RedisCacheDataModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ChatApplication.ViewModels
+{
+    public class RedisCacheDataModel
+    {
+        public string UserName { get; set; }
+        public string Text { get; set; }
+        public DateTime When { get; set; }
+
+        public RedisCacheDataModel()
+        {
+            When = DateTime.Now;
```

Comment:
Why did you add here this constructor? What is its mission?

Reply:
This constructor  initializes time in realtime